clicc_devices/templates/view_sets.html

```diff
@@ -0,0 +1,36 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block content %}
+<div class="container mt-4">
+    <h1 class="mb-4">Sets</h1>
+    <div class="table-responsive">
+        <table class="table table-hover table-striped table-bordered align-middle">
+            <thead>
+                <tr>
+                    <th scope="col">Set Name</th>
+                    <th scope="col">Unit</th>
+                    <th scope="col">Type</th>
+                    <th scope="col">Last Retrieved</th>
+                    <th scope="col">Item Count</th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for set in set_data %}
+                <tr>
+                    <td>{{ set.name }}</td>
+                    <td>{{ set.unit }}</td>
+                    <td>{{ set.type }}</td>
+                    <td>{{ set.retrieved }}</td>
+                    <td>{{ set.item_count }}</td>
```

Comment:
I think these numbers should be right-aligned.  I don't see an appropriate [template filter](https://docs.djangoproject.com/en/5.2/ref/templates/builtins/) - maybe `rjust`? - CSS might be better.